lib/ui/forgot_password/forgot_password_screen.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:survey_flutter_ic/gen/assets.gen.dart';
+import 'package:survey_flutter_ic/ui/widget/input_field_widget.dart';
+import 'package:survey_flutter_ic/utils/dimension.dart';
+
+class ForgotPasswordScreen extends StatelessWidget {
+  const ForgotPasswordScreen({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        backgroundColor: Colors.transparent,
+        body: Container(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage(Assets.images.bgLoginOverlay.path),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Padding(
+            padding:
+                const EdgeInsets.symmetric(horizontal: Dimensions.paddingLarge),
+            child: Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceAround,
+                      children: [
+                        _nimbleLogo,
+                        const SizedBox(height: 24.0),
+                        _forgotPasswordInstruction(context),
+                        const SizedBox(height: 100.0),
+                      ]),
+                  _emailTextField,
+                  const SizedBox(height: Dimensions.paddingMedium),
+                  ElevatedButton(
+                    onPressed: () {},
+                    style: ElevatedButton.styleFrom(
+                      minimumSize:
+                          const Size.fromHeight(Dimensions.buttonHeight),
+                      backgroundColor: Colors.white,
+                      shape: RoundedRectangleBorder(
+                        borderRadius:
+                            BorderRadius.circular(Dimensions.radiusNormal),
+                      ),
+                    ),
+                    child: Text(
+                      AppLocalizations.of(context)!.forgotPasswordResetButton,
+                      style: Theme.of(context)
+                          .textTheme
+                          .titleSmall
+                          ?.copyWith(color: Colors.black),
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          ),
+        ));
+  }
+
+  Widget _forgotPasswordInstruction(BuildContext context) => Text(
+        AppLocalizations.of(context)!.forgotPasswordInstruction,
+        style: Theme.of(context)
+            .textTheme
+            .bodyLarge
+            ?.copyWith(color: Colors.white70),
+        textAlign: TextAlign.center,
+      );
+
+  Widget get _nimbleLogo => Image(
+        image: Assets.images.icNimbleWhiteLogo.image().image,
+        width: 168.0,
+        height: 40.0,
+      );
```

Comment:
Personally, I don't think the image `nimble` should have fixed width and height values as in different screen sizes, it would keep the same size and may display too small or too big

I think we should create a Padding box for it and the image should fitWidth with that box while remaining the height ratio for more flexible layout

```suggestion
  Widget get _nimbleLogo => Padding(
      padding: const EdgeInsets.only(left: 104.0, right: 104.0),
      child: Image.asset(
        Assets.images.icNimbleWhiteLogo.path,
        fit: BoxFit.fitWidth,
      ));
```